test/functional/test_framework/test_node.py

```diff
@@ -333,9 +342,9 @@ def stop_node(self, expected_stderr='', *, wait=0, wait_until_stopped=True):
         try:
             # Do not use wait argument when testing older nodes, e.g. in feature_backwards_compatibility.py
             if self.version_is_at_least(180000):
-                self.stop(wait=wait)
+                self.node_exit(wait=wait)
```

Comment:
Kind of nitty, but I'd rather call it `node_shutdown` or `node_stop`.

Reply:
Changed